Scripts/script-ADGetUser.yml

```diff
@@ -50,7 +75,23 @@ script: |-
       except Exception as ex:
           return { 'Type' : entryTypes['error'], 'ContentsFormat' : formats['text'], 'Contents' : 'Error occurred while parsing output from ad command. Exception info: ' + str(ex) + '\nInvalid output:\n' + str( resAD ) }
 
-  def listAll(attrs, using):
+  def translateUserAccountControl(resAD, userAccountControlOut):
+      try:
+          for m in resAD:
+              if isError(m):
+                  continue
+              if 'UserAccountControl' not in m['Contents'][0] or m['Contents'][0]['UserAccountControl'] == "":
```

Comment:
Use `demisto.get` to extract the value. if `m['Contents'][0]` doesn't exists, it will prompt an error